docs/janssen-server/install/setup.md

```
@@ -99,66 +99,66 @@ Use this script to configure your Jans Server and to add initial data. If setup.
+| -admin-password ADMIN_PASSWORD | Used as the Administrator password                                                                        |
+| -jans-max-mem JANS_MAX_MEM | Total memory (in KB) to be used by Janssen Server                                                         |
+| -properties-password PROPERTIES_PASSWORD | Encoded setup.properties file password                                                                    |
+| -approved-issuer APPROVED_ISSUER | Api Approved Issuer                                                                                       |
+| --force-download | Force downloading files                                                                                   |
+| --download-exit | Download files and exits                                                                                  |
+| -jans-app-version JANS_APP_VERSION | Version for Janssen applications                                                                          |
+| -jans-build JANS_BUILD | Build version for Janssen applications                                                                    |
+| -setup-branch SETUP_BRANCH | Janssen setup GitHub branch                                                                               |
+| --disable-config-api-security | Turn off oauth2 security validation for jans-config-api                                                   |
+| --cli-test-client | Use config api test client for CLI                                                                        |
+| --import-ldif IMPORT_LDIF | Render ldif templates from directory and import them in Database                                          
```

markdownlint-cli2 (0.22.1)

[warning] Table pipe style
Expected: leading_and_trailing; Actual: leading_only; Missing trailing pipe

(MD055, table-pipe-style)